lib/commands/apps.js

```diff
@@ -0,0 +1,17 @@
+let extensions = {}, commands = {};
+
+/**
+ *  Gets system and installed applications info
+ *  The information consists of :-
+ *   - packageName
+ *   - appName
+ *   - packageActivty
+ * */
+
+commands.mobileGetAppsInfo = async function () {
+    return await this.jwproxy.command('/appium/device/apps', 'GET', {});
```

Comment:
please test your PRs before publishing them. This call won't work as jwproxy only supports in-session endpoints and the `session/<id>` prefix will be automatically added to it

Reply:
Yes got it